doc/specs/stdlib_strings.md

```diff
@@ -192,3 +192,64 @@ program demo
   print'(a)', ends_with("pattern", "pat")  ! F
 end program demo
 ```
+
+
+<!-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -->
+### `slice`
+
+#### Description
+
+Extracts the characters from the defined region of the input string.  
+Argument `first` and `last` defines the region for the function `slice` to operate.  
+Extraction starts from the index `first` and takes stride of length `stride`.  
+Argument `stride` cannot take the value 0.
```

Comment:
Might be worth stating explicitly, that in case stride is 0, the function uses the default value 1 instead.